frontend/src/bundles/auth/components/AdditionalForm.vue

```diff
@@ -1,15 +1,51 @@
 <template>
-  <form :value="!invalid" @submit.prevent="handleSubmit">
+  <form
+    :value="!invalid"
+    @submit.prevent="handleSubmit"
+  >
+    <validation-provider
+      name="gender"
+    >
+      <p>You are <span class="primary--text font-weight-bold">{{ gender || 'male' }}</span></p>
+      <v-radio-group
+        v-model="gender"
+        row
+      >
+        <v-radio
+          v-for="(gender, index) in genders"
+          :key="index"
+          :label="gender.gender_label"
+          :value="gender.gender_name"
+        />
+      </v-radio-group>
+    </validation-provider>
+    <validation-provider
+      name="gender_preference"
+    >
+      <p>Your preference selection <span class="orange--text font-weight-bold">{{ gender_preference || 'female' }}</span></p>
```

Comment:
The same 
You already have default value

Reply:
fixed